Europa/Österreich/Schweiggers.md

```diff
@@ -0,0 +1,26 @@
+*Hier sieht man wie ein Beispielreiseführereintrag aussehen sollte.*
+
+# Schweiggers
+
+Schweiggers liegt im Waldviertel in Niederösterreich. Die Fläche der Marktgemeinde umfasst 58,65 Quadratkilometer. 25,04 Prozent der Fläche sind bewaldet.
+
+Zwischen den Ortsteilen Mödershöf und Schweiggers entspringt die Deutsche Thaya, einer von zwei Quellflüssen der Thaya.
+
+## Geschichte
+
+Der Name Schweiggers leitet sich vom althochdeutschen Vornamen Swidger ab und taucht in den Urkunden erstmals um 1182 als „Swichers“ auf. Der Name veränderte sich im Laufe der Jahrhunderte, bis er 1643 als „Schweiggers“ aufscheint.
+
+1394 erhielt die Gemeinde das Marktrecht und hat dadurch einen weitläufigen Marktplatz im Zentrum des Ortes.
+
+1966 wurde die Marktgemeinde Schweiggers mit den Gemeinden Mannshalm (mit den Katastralgemeinden Mannshalm, Streitbach Schwarzenbach und Meinhartsschlag) und Perndorf (mit den KGs Perndorf, Kleinwolfgers und Unterwindhag) zusammengelegt.
+
+1971 erfolgte die zweite Zusammenlegung mit den Gemeinden Sallingstadt (KG Sallingstadt und Walterschlag) sowie Limbach (KG Limbach und Windhof) und Siebenlinden (KG Siebenlinden, Vierlings, Brunnhöf, Bichlhof, Groß Reichenbach und Reinbolden).
+
+## Sehenswürdigkeiten
+
+* Waldviertler Erdäpfelwelt
+* Pfarrkirche Siebenlinden
+
+## Bild
+
+<img src="https://de.wikipedia.org/wiki/Schweiggers#/media/Datei:Rathaus_in_Schweiggers_2018.jpg" alt="Rathaus in Schweiggers\>
```

Comment:
Hier ist ein kleiner Fehler, wodurch nur die Codezeile und nicht das Bild angezeigt wird.
Am ende muss ein Slash und kein Backslash stehen.

Reply:
Auch ich habe Probleme mit dem Bild festgestellt. Bei mir lädt das Bild über die folgende FileURL: "https://upload.wikimedia.org/wikipedia/commons/c/c5/Rathaus_in_Schweiggers_2018.jpg" Versuche diese mal einzubinden in deine Datei.

Reply:
Ich habe die URL geändert.